lib/linguist/languages.yml

```diff
@@ -1309,6 +1309,16 @@ DataWeave:
   ace_mode: text
   tm_scope: source.data-weave
   language_id: 974514097
+DenizenScript:
+  type: programming
+  color: "#FBEE96"
+  ace_mode: yaml
+  codemirror_mode: yaml
+  codemirror_mime_type: text/x-yaml
+  extensions:
+  - ".dsc"
```

Comment:
This is going to cause misidentification a ton of files as this extension is also used by Debian for source control files. There are waaaay more of these on GitHub than DenizenScript so you'll need to add support for these too with at least two samples.

Reply:
Would it be possible to simply just write some heuristics for DenizenScript to avoid the matching of Debian source control files, or does Linguist not work that way?
If not I can always *try* to add the Debian source files.

Reply:
Hmm I see. Well, I guess I can try to add support for the Debian files. Might be the wrong person to do it as I don't use a Linux operating system, but I'll give it a try! 😁

Reply:
Should I add the Debian source file support in this PR or should I open a second PR? Just want to make sure.

Reply:
This PR is fine.